### ISSUE TITLE
Gsettings/dconf option to show/hide power off, lock screen?

### ISSUE THREAD

**Issue by IOMadness** [getsolus/brisk-menu#38]
I'm running MATE on RHEL 7 and was trying to configure a way to lockdown the system from users hitting the lock screen or power off buttons. I've found a way to disable the effects of clicking the buttons, rendering them essentially useless, but was seeing if there was an option to actually show/hide the power off and lock screen buttons in Brisk Menu so that users don't even have the temptation to do so.

I've checked the available keys through gsettings, which show:
com.solus-project.brisk-menu label-visible
com.solus-project.brisk-menu pinned-shortcuts
com.solus-project.brisk-menu favourites
com.solus-project.brisk-menu rollover-activate
com.solus-project.brisk-menu dark-theme
com.solus-project.brisk-menu hot-key
com.solus-project.brisk-menu label-text
com.solus-project.brisk-menu search-position
com.solus-project.brisk-menu window-type

But none of these seem to address this issue. Is there a way to do this? Thanks!

**Comment by BoxFoxCode** [comment]
No, there is no setting to do that for the classic window type menu.
Since you broke their function though; a workaround for your needs may be, you could theme them to be invisible (0 opacity). Of course they're still there, and clickable if you had not interrupted their function however you did. However, adding a new theme would need such styling appended. (Hence why a "workaround.")
The session buttons have the style class `session-button` .
```css
.brisk-menu .session-button {
}
```
---
I could see how this feature/toggle could be useful though otherwise. For styling/reducing UI redundancy if someone uses another applet for session controls, or any other setup of session controls.

**Comment by IOMadness** [comment]
I'll revisit and see if we can implement this on our setup. Our use case is that we allow our users to get on our HPC via FastX where the user can also select their preferred desktop (GNOME, LxQT, Mate, Xfce, etc), but users manage their sessions through FastX, and disable their use of power off/shutdown/suspend.

Thanks for getting back!
